datarobot_batch_scoring/network.py

```diff
@@ -169,8 +169,8 @@ def _response_callback(self, r, batch=None, *args, **kw):
                 self.ui.error(msg)
                 self.send_error_to_ctx(batch, msg)
             else:
-                self.ui.warning('respooling batch {}'
-                                .format(batch.id))
+                self.ui.warning('retrying failed batch {}, attempts left: {}'
```

Comment:
fun fact: batch scoring's `UI` class doesn't support this: https://github.com/datarobot/batch-scoring/blob/87148321b07c16d9b34d6818f5c89276415d75eb/datarobot_batch_scoring/utils.py#L126-L126,
it accepts just a string and no additional args. 

but you're absolutely right.